exporter/collector/internal/datapointstorage/datapointcache.go

```diff
@@ -81,7 +81,7 @@ func (c Cache) gc(shutdown <-chan struct{}, tickerCh <-chan time.Time) bool {
 }
 
 // Identifier returns the unique string identifier for a metric
-func Identifier(resource *monitoredrespb.MonitoredResource, extraLabels map[string]string, metric pdata.Metric, labels pdata.AttributeMap) string {
+func Identifier(resource *monitoredrespb.MonitoredResource, extraLabels map[string]string, metric pdata.Metric, labels pdata.Map) string {
```

Comment:
same nit here, OTel used to call them labels and now they're "attributes". Might be worth updating, but up to you

Reply:
done